From 2ed727b4136ca3a4ce25782357c16da642573757 Mon Sep 17 00:00:00 2001
From: Florian Fuchs <flfuchs@redhat.com>
Date: Mon, 15 Aug 2016 15:38:23 +0200
Subject: [PATCH] Exclude more unneeded files from default plan

This patch exludes more file types from the tarball uploaded to swift as
the default deployment plan.

Change-Id: I8b6d8de8d7662604cdb871fa6a4fb872c7937e25
Closes-Bug: #1613286
---

diff --git a/tripleo_common/utils/tarball.py b/tripleo_common/utils/tarball.py
index 147fa7e..20523ec 100644
--- a/tripleo_common/utils/tarball.py
+++ b/tripleo_common/utils/tarball.py
@@ -23,7 +23,10 @@
     """Create a tarball of a directory."""
     LOG.debug('Creating tarball of %s at location %s' % (directory, filename))
     processutils.execute('/usr/bin/tar', '-C', directory, options, filename,
-                         '--exclude', '.git', '--exclude', '.tox', '.')
+                         '--exclude', '.git', '--exclude', '.tox',
+                         '--exclude', '*.py', '--exclude', '*.pyc',
+                         '--exclude', '*.pyo', '--exclude', '*.pp',
+                         '--exclude', '*.sh', '.')
 
 
 def tarball_extract_to_swift_container(object_client, filename, container):
